src/hiveboard_bridge/src/main.cpp

```diff
@@ -27,8 +32,23 @@ int getTcpServerPort() {
     return port;
 }
 
+std::string getRobotName() {
+    std::string robotName;
+
+    if (!ros::param::get("~ROBOT_NAME", robotName)) {
+        ROS_INFO("No ROBOT_NAME parameter was given. Using default value %s", DEFAULT_ROBOT_NAME);
```

Comment:
I'm pretty sure you can put a default value for ros when getting param, see
http://wiki.ros.org/roscpp/Overview/Parameter%20Server#Getting_Parameters